### PR TITLE
feat: new page for functions

### LINKED ISSUE #888: Revamp of Functions page and related pages
They need:
* Better formatting
* More thorough description of attributes (`extends`, `mutates`, etc.)
* Mention of `receive()`, `external()`, `bounced()` with links to relevant pages
* Consistent and cohesive descriptions
* Better cross-linkage
* ...

Separate receivers and getters from all other functions — make two groups. Or three, to put getters in their own category.

### PR DESCRIPTION
**Pull request by novusnota**
WIP: WIP

Closes tact-lang/tact#888

### CODE REVIEW

File: pages/book/functions.mdx [hunk 1]
```diff
@@ -23,47 +29,99 @@ fun bar() {
...
 
-## Global static functions
+## Native functions, `native` [#native]
+
+{/* TODO Top-level, mappings to FunC */}
+
+Native functions are direct bindings of FunC functions:
+
+> **Note**
+> Native functions could be also mutable and extension ones.
```
Comment (anton-trunov) [inline pages/book/functions.mdx:39]:
```suggestion
> Native functions could be also mutating and extension ones.
```

Comment (novusnota) [reply pages/book/functions.mdx:39]: Thanks! Although I've just posted a very rough draft, and this paragraph will be annihilated and then brought back from the ashes :)

**Comment by anton-trunov** [comment]
Any progress here?

**Comment by anton-trunov** [comment]
Any progress here?

**Comment by anton-trunov** [comment]
we need this page asap, especially with enhanced section on native functions showing how to FFI with FunC

**Comment by anton-trunov** [comment]
for instance, here is an example of a binding for FunC's `parse_std_addr` function:

```
;; FunC
(int, int) parse_std_addr(slice s) asm "REWRITESTDADDR";
```


```
// Tact
struct Addr {
    workchain: Int;
    address: Int;
}

@name(parse_std_addr)
native parseStdAddr(s: Slice): Addr;
```

(caveat emptor: I haven't tested it)

**Comment by anton-trunov** [comment]
Any progress here?

**Comment by novusnota** [comment]
Closed in favor of opening a new PR in https://github.com/tact-lang/tact

The commits here should be kept for a while, so the branch must not be deleted!

**Pull request closed without merging**